specification/security/resource-manager/Microsoft.Security/stable/2017-08-01/complianceResults.json

```diff
@@ -0,0 +1,169 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Center",
+    "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+    "version": "2017-08-01"
+  },
+  "host": "management.azure.com",
+  "schemes": ["https"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "security": [{
+    "azure_auth": ["user_impersonation"]
+  }],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/{scope}/providers/Microsoft.Security/complianceResults": {
```

Comment:
>complianceResults [](start = 43, length = 17)

You should declare paging via the x-ms-pageable extension even if your service doesn't support paging right now. It will save you some back-compat pain later on.